package.json

```diff
@@ -22,6 +22,7 @@
   "author": "Procore Tech <insights@procore.com> (http://procore.com)",
   "license": "MIT",
   "dependencies": {
+    "@procore/js-sdk": "^2.4.1",
```

Comment:
Why is this here? Why would it have a dependency on itself?

Reply:
I have removed these changes in addition to `yarn.lock`. It seems that the tests need the sdk in `./dist` and to accomplish that I ran `yarn add @procore/js-sdk` which in turn adds this sdk as a dep of itself.